index.js

```diff
@@ -0,0 +1,71 @@
+const https = require('https');
+
+const token = process.env.CIRCLECI_TOKEN;
+const target_repo = process.env.TARGET_REPO;
+const target_job = process.env.TARGET_JOB;
+const ignore_branches = process.env.IGNORE_BRANCHES.split(',');
+
+function requestCircleCIBuild(repo, branch, callback) {
+  const postData = {
+    build_parameters: {
+      CIRCLE_JOB: target_job,
+    },
+  };
+  const postDataStr = JSON.stringify(postData);
+  const options = {
+    hostname: 'circleci.com',
+    port: 443,
+    path: `/api/v1.1/project/github/${repo}/tree/${branch}`,
+    method: 'POST',
+    auth: `${token}:`,
+    headers: {
+      'Content-Type': 'application/json',
+      'Content-Length': postDataStr.length,
+    },
+  };
+  const req = https.request(options);
+
+  req.on('error', (e) => {
+    console.error(e);
+    callback(e);
+  });
+
+  req.write(postDataStr);
+  req.end(callback);
+}
+
+function processEvent(event, callback) {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
  Expected to return a value at the end of function 'processEvent'  consistent-return